src/algokit/cli/doctor.py

```diff
@@ -35,7 +36,15 @@
     is_flag=True,
     default=False,
 )
-def doctor_command(*, copy_to_clipboard: bool) -> None:
+@click.option(
```

Comment:
AlgoKit already support a verbose flag at the root command, so it would be possible for a user to run `algokit -v doctor -v` which feels a little strange. Could/Should we detect the `-v` flag at the `algokit` level and activate instead?

@lempira was also talking about potentially leveraging doctor to expose project package information when run inside an AlgoKit project. We probably want to consider how that'd impact a change like this.

I'm wondering if this feature should be extracted into a seperate PR outside the fix, so we can consider a few other facets?

Reply:
As discussed on a call, this doesnt add any new commands groups options or arguments to doctor (i will reuse global verbose flag instead) - all it changes is appending extra info that can be super helpful when debugging user queries. If @lempira would be adding new functionality to doctor it would require changes regardless + would potentially be handled via subcommand group, so i don't this will complicate this or overlap with potential new behaviour that would be introduces